gemma-core/src/test/java/ubic/gemma/core/search/SearchServiceTest.java

```diff
@@ -143,7 +144,7 @@ public void tearDown() {
      * are found, -- requires LARQ index.
      */
     @Test
-    public void testGeneralSearch4Brain() throws SearchException {
+    public void testGeneralSearch4Brain() throws SearchException, OntologySearchException {
```

Comment:
This should not be necessary, the search service does not throw OntologySearchException.

Reply:
It's actually used with OntologyService, everything is ok!